package.json

```diff
@@ -124,9 +124,9 @@
     "@types/rimraf": "^0.0.28",
     "@types/semver": "^7.3.9",
     "@types/shelljs": "^0.8.5",
-    "@typescript-eslint/parser": "7.11.0",
-    "@typescript-eslint/rule-tester": "7.11.0",
-    "@typescript-eslint/utils": "7.11.0",
+    "@typescript-eslint/parser": "8.0.0-alpha.20",
+    "@typescript-eslint/rule-tester": "8.0.0-alpha.20",
+    "@typescript-eslint/utils": "8.0.0-alpha.20",
```

Comment:
I upgraded these to be compatible with Angular ESLint.
Otherwise the linting rules would work, but there might be a TS type error/warning within the eslint configuration file.

Because of the upgrade some stylistic rules were removed (e.g. spacing and quotes).
If we want to re-enable this we have to install the ESLint Stylistic Plugin, for more info see  https://eslint.style/guide/why